### PR TITLE
Fix `XorK` call graph, remove control_val

### PR DESCRIPTION
**Pull request by anurudhp**

### CODE REVIEW

File: qualtran/bloqs/arithmetic/bitwise.py [hunk 1]
```diff
@@ -50,33 +47,23 @@ def _cvs_converter(vv):
 
 
 @frozen
-class XorK(SpecializedSingleQubitControlledGate):
+class XorK(Bloq):
```
Comment (tanujkhattar) [inline qualtran/bloqs/arithmetic/bitwise.py:50]: Do we need the `_cvs_converter` defined above? It doesn't seem to be used in this file

Comment (tanujkhattar) [inline qualtran/bloqs/arithmetic/bitwise.py:50]: While we are at it, let's also override the `wire_symbol` method for both the bloqs in this file and return `TextBox(f"⊕ {K}")` for the `XorK` and `TextBox(f"x⊕y")` for `Xor` for the target registers

**Review by tanujkhattar — APPROVED** [review]
LGTM % comments

File: qualtran/bloqs/arithmetic/bitwise.py (hunk 1 above)
Comment (anurudhp) [reply qualtran/bloqs/arithmetic/bitwise.py:45]: added wire symbols and a circuit diagram unit test

Comment (anurudhp) [reply qualtran/bloqs/arithmetic/bitwise.py:45]: oops, removed

**Pull request merged**